share/man/man4/ure.4

```diff
@@ -28,7 +30,7 @@
 .Os
 .Sh NAME
 .Nm ure
-.Nd "RealTek RTL8152/RTL8153/RTL8153B/RTL8156/RTL8156B USB to Ethernet controller driver"
+.Nd RealTek RTL8152/RTL8153/RTL8156 USB Ethernet driver
```

Comment:
Removed use of quoted literals because it does nothing, therefore confuses people.